Say/ViewController.swift

```diff
@@ -131,35 +131,24 @@ class ViewController: NSViewController {
     
     @IBAction func selectText(_ sender: NSTextField) {
         if let url = URL(string: sender.stringValue) {
+            let instapaper = "https://www.instapaper.com/text?u="
+            let onlyText = URL(string:"\(instapaper)\(url)")
             // if URL format is right
-            if let data = NSData.init(contentsOf: url) {
+            if let data = NSData(contentsOf: onlyText!) {
```

Comment:
Move `!` operator to upper line where the URL is built.

About the operator `!`, it casts optional type to value type. So in this case, `onlyText` is `URL?`. But if you are sure the URL is always not-null, the variable is better to be value there as `URL` and it doesn't need to be casted in if-statement.